packages/app-store/index.ts

```diff
@@ -22,6 +22,8 @@ import * as sendgrid from "./sendgrid";
 import * as stripepayment from "./stripepayment";
 import * as tandemvideo from "./tandemvideo";
 import * as vital from "./vital";
+//TODO change dirName if needed
```

Comment:
I don't think we need to change that. So, we can remove this TODO.

Reply:
done